acceptance/openstack/identity/v3/groups_test.go

```diff
@@ -62,6 +62,62 @@ func TestGroupCRUD(t *testing.T) {
 		tools.PrintResource(t, g.Extra)
 	}
 
+	var found bool
+	for _, group := range allGroups {
+		tools.PrintResource(t, group)
+		tools.PrintResource(t, group.Extra)
+
+		if group.Name == "testgroup" {
```

Comment:
The test is failing here. The name is randomly generated, such as `ACPTTESTkl7t28j7`. My suggestion would be to  do

```go
if group.Name == newGroup.Name {
```